packages/rich-text/README.md

```diff
@@ -14,169 +14,507 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 ## Usage
 
-### create
+<!-- START TOKEN(Autogenerated API docs) -->
 
-```js
-create( {
-	?element: Element,
-	?text: string,
-	?html: string,
-	?range: Range,
-	?multilineTag: string,
-	?multilineWrapperTags: Array,
-} ): Object
-```
+### apply
```

Comment:
This one should ideally also be marked unstable, as it's DOM-specific. It was exported so that the `RichText` component in the editor package could make use of it, but I consider this an internal function.

Reply:
Added `apply` to the ignore list at 3d61f40